www/form/inner.js

```diff
@@ -1995,6 +2004,75 @@ define([
             },
             icon: h('i.cptools.cptools-form-grid-radio')
         },
+        date: {
+        defaultOpts: {
+            type: 'date',
+        },
+        get: function (opts, a, n, evOnChange) {
+            opts = Util.clone(TYPES.date.defaultOpts)
+            var tag = h('input', {
+                type: opts.type,
+            });
+
+            var $tag = $(tag);
+            $tag.on('change keypress', Util.throttle(function () {
+                evOnChange.fire();
+            }, 500));
+            var cursorGetter;
+            var setCursorGetter = function (f) { cursorGetter = f; };
+
+            return {
+                tag: tag,
+                isEmpty: function () { return !$tag.val().trim(); },
+                getValue: function () {
+                    var invalid = $tag.is(':invalid');
+                    if (invalid) { return; }
+                    return $tag.val();
+                    
+                },
+                setValue: function (val) { $tag.val(val); },
+                setEditable: function (state) {
+                    if (state) { $tag.removeAttr('disabled'); }
+                    else { $tag.attr('disabled', 'disabled'); }
+                },
+                edit: function (cb, tmp) {
+
+                    return editDateOptions(cb);
+                },
+                getCursor: function () { return cursorGetter(); },
+                reset: function () { $tag.val(''); }
+            };
+        },
+        printResults: function (answers, uid) { // results text
+            var results = [];
+            var empty = 0;
+            var tally = {};
+
+            var isEmpty = function (answer) {
+                return !answer || !answer.trim();
+            };
+
+            Object.keys(answers).forEach(function (author) {
+                var obj = answers[author];
+                var answer = obj.msg[uid];
+                if (isEmpty(answer)) { return empty++; }
+                Util.inc(tally, answer);
+            });
+
+            //if (max < 2) { // there are no duplicates, so just return text
+                results.push(getEmpty(empty));
+                Object.keys(answers).forEach(function (author) {
+                    var obj = answers[author];
+                    var answer = obj.msg[uid];
+                    if (!answer || !answer.trim()) { return empty++; }
+                    results.push(h('div.cp-charts-row', h('span.cp-value', answer)));
+                });
+                return h('div.cp-form-results-contained', h('div.cp-charts.cp-text-table', results));
+
+
+            return h('div.cp-charts.cp-bar-table', results);
+        },
+        icon: h('i.cp-calendar-active.fa.fa-calendar')},
```

Comment:
this line is causing UI issues
- the class `cp-calendar-active` is only used in the calendar app, not needed here
- the icon is not aligned properly in the UI (see on hover) which could be due to the fact that it's the only icon to use `fa` while others use `cptools` there might be a CSS selector to correct